modules/oxide/src/less/theme/components/collection/collection.less

```diff
@@ -157,7 +157,7 @@
     color: @collection-grid-item-label-enabled-text-color;
   }
 
-  .tox-collection--grid .tox-collection__item--active {
+  .tox-collection--grid .tox-collection__item--active:not([aria-disabled=true]) {
```

Comment:
Should this be using `:not(.tox-collection__item--state-disabled)` as that's what everything else uses? Also could this be merged in with the following rule?

```
  .tox-collection--grid .tox-collection__item--active:not(.tox-collection__item--state-disabled) {
    color: @collection-grid-item-label-active-text-color;
  }
```
```

Reply:
I have made this change but I have asked @lostkeys if there is any reason why the two rules shouldn't be merged.